make.sh

```diff
@@ -527,6 +527,13 @@ pkg_install_rust() {
     _fold_end
 }
 
+pkg_install_solc_linux() {
```

Comment:
rename to pkg_install_solc() instead since our default pkg manager is linux-based os.

Reply:
Fixed: 45f8c29ce87c91e714ab0edb9f654a265531868b